src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Quadrangle {
+    public IsoscelesTrapezoid(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: isoscelesTrapezoid");
+        super.draw();
+    }
```

Comment:
In the `draw` method, the message to be printed should include all the information about the trapezoid, such as color, width, and height, to match the expected behavior. Also, instead of `System.out.print`, it's better to use `System.out.println` for proper output formatting.